dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1112,19 +1113,32 @@ export const LiveLayout = ({
 									`}
 								>
 									<RightColumn>
-										{renderAds && (
-											<AdSlot
-												position="right"
-												display={format.display}
-												shouldHideReaderRevenue={
-													article.shouldHideReaderRevenue
-												}
-												isPaidContent={
-													article.pageType
-														.isPaidContent
-												}
-											/>
-										)}
+										{renderAds ? (
+											format.design ===
+												ArticleDesign.LiveBlog ||
+											format.design ===
+												ArticleDesign.DeadBlog ? (
```

Comment:
I think this condition is always true on live layouts? (see [DecideLayout.tsx](https://github.com/guardian/dotcom-rendering/blob/e65e2d33014a7b1a330fa8ba50c396a51dd314fe/dotcom-rendering/src/layouts/DecideLayout.tsx#L90-L91)).

Reply:
Ah yeah, nice one. I guess that LiveLayout means liveblog!